spec/features/conditions/user_can_update_conditions_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../../spec_helper'
+
+describe "Conditions/id/edit" do
```

Comment:
we'll want to describe specific user stories, not just the path name for a specific action